helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -104,8 +104,87 @@ receivers:
             service.instance.id:
               enabled: false
           {{- else }}
-          metrics_path: '`"prometheus.io/path" in annotations ? annotations["prometheus.io/path"] : "/metrics"`'
-          endpoint: '`endpoint`:`"prometheus.io/port" in annotations ? annotations["prometheus.io/port"] : 9090`'
+          config:
+            scrape_configs:
+              - job_name: 'istio'
+                metrics_path: '`"prometheus.io/path" in annotations ? annotations["prometheus.io/path"] : "/metrics"`'
+                scrape_interval: 10s
+                static_configs:
+                  - targets: ['`endpoint`:`"prometheus.io/port" in annotations ? annotations["prometheus.io/port"] : 9090`']
+                metric_relabel_configs:
+                  - source_labels: [__name__]
+                    action: keep
+                    regex: "(envoy_cluster_lb_healthy_panic|\
+                    envoy_cluster_manager_warming_clusters|\
+                    envoy_cluster_membership_healthy|\
+                    envoy_cluster_membership_total|\
+                    envoy_cluster_ssl_handshake|\
+                    envoy_cluster_ssl_session_reused|\
+                    envoy_cluster_ssl_versions_TLSv1_2|\
+                    envoy_cluster_ssl_versions_TLSv1_3|\
+                    envoy_cluster_upstream_cx_active|\
+                    envoy_cluster_upstream_cx_close_notify|\
+                    envoy_cluster_upstream_cx_connect_attempts_exceeded|\
+                    envoy_cluster_upstream_cx_connect_ms|\
```

Comment:
The name filtering is all copied from [istio discovery](https://github.com/signalfx/splunk-otel-collector/blob/4bf9ff53d15faef23424ac028e5a0f1c0cf23eb8/cmd/discoverybundler/metadata/receivers/istio.yaml#L7) except for this line. All of the metrics are also being checked for the optional histogram suffixes of `_sum`, `_count`, and `_bucket`. When this metric has `_sum` in the filter (as it does in istio discovery linked above), the filtering doesn't match and all of the metrics with this name are dropped. Removing `_sum` properly matches the metric.